django_project/realtime/tasks/test/test_realtime_tasks.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+"""Docstring here."""
+import os
+import unittest
+
+from django import test
+
+from realtime.tasks.realtime.flood import process_flood
+from realtime.tasks.flood import create_flood_report
+from realtime.tasks.realtime.celery_app import app as realtime_app
+
+flood_data_path = '/home/realtime/hazard-drop/flood_data.json'
```

Comment:
Put this inside the unittests.

Reply:
ah, so we can use file from outside hazard-drop directory?